compiler/record-minmax-conversion-test/testall.sh

```diff
@@ -25,6 +25,20 @@ PASSED=()
 FAILED=()
 
 pushd "${WORKDIR}"
+
+# Generate h5 input data
+source "${VIRTUALENV}/bin/activate"
+  "${VIRTUALENV}/bin/python" "${GEN_SCRIPT_PATH}" \
+  --num_data 3 \
+  --output_dir "${BIN_PATH}" \
+  --artifact_dir ${WORKDIR} \
+  --model "$@"
+
+if [[ $? -ne 0 ]]; then
+  echo "FAILED TO GENERATE INPUT"
+  continue
```

Comment:
This is not in the loop anymore.

```suggestion
  exit 255
```